docs/event-schemas/wfm/WorkflowRunStateChange.json

```diff
@@ -0,0 +1,104 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "WorkflowRunStateChange"
+  },
+  "paths": {},
+  "components": {
+    "schemas": {
+      "AWSEvent": {
+        "type": "object",
+        "required": [
+          "detail-type",
+          "detail",
+          "id",
+          "source",
+          "time"
```

Comment:
Should we make `account` mandatory? It's used in the archive filter.
Also, latest once we fix the payload schema(s), we should make the version mandatory.

Reply:
Yup. We can do that.

Reply:
Final AWS Envelope:

We did run experiment and, turned out `id` and `version` are not in our control. EventBus fills these value regardless. Also, `version` seems to be always set to `0` on the actual event instance that sent into the channel; though we payload it with `1.0.0`. 

The `account`, `region`, `time` observed to be auto-filled by EventBridge as well.

Hence. We concluded that we will be just doing the required fields on `detail`, `detail-type`, `source` as per AWS documentation. See below.

Minimum information needed for a valid custom event
https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events-structure.html

Reply:
And some more reading on this:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/ev-events.html
The version seems applied in "Create a schema from events on an event bus"
>If the contents of events on the event bus change, EventBridge creates new versions of the related EventBridge schema.
https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-schema-create.html

Reply:
Updated with https://github.com/umccr/orcabus/commit/1e903c67c1eb0fe8ffa949171a058c93c835c4d7

Reply:
Also harmonised/updated SRSC envelope with https://github.com/umccr/orcabus/commit/34904419b98e20498a02a65c685c93f2ba93993d